testutils/utils.go

```diff
@@ -136,11 +147,8 @@ func testPutGetDeleteExists(t *testing.T, kv store.Store) {
 
 func testWatch(t *testing.T, kv store.Store) {
```

Comment:
I am really not sure what to do here... sending a sequence of value fixes the test on Consul and Zookeeper, but on etcd updates seem to come out of order :(